dev-tools/phpstan/baseline.php

```diff
@@ -572,7 +572,7 @@
 	'path' => __DIR__ . '/../../src/Fixer/PhpUnit/PhpUnitDedicateAssertFixer.php',
 ];
 $ignoreErrors[] = [
-	'message' => '#^Parameter \\#1 \\$token of class PhpCsFixer\\\\Tokenizer\\\\Token constructor expects array\\{int, string\\}\\|string, array\\{262, int\\|string\\|true\\} given\\.$#',
+	'message' => '#^Parameter \\#1 \\$token of class PhpCsFixer\\\\Tokenizer\\\\Token constructor expects array\\{int, string\\}\\|string, array\\{[0-9]+, int\\|string\\|true\\} given\\.$#',
```

Comment:
Anyone running `composer phpstan:baseline` would have this reverted to `262` or, in few hours/days to `313`.

Reply:
Yeah, you're right, it would make sense only in `ignoreErrors` section in the main config. But it would be great if PHPStan was smarter here and used `[0-9]+` for `T_*` in regexes 🙂.